package.json

```diff
@@ -4,7 +4,13 @@
   "description": "fetch for read and write access to the local file system",
   "main": "index.js",
   "scripts": {
-    "test": "standard && mocha"
+    "test": "standard && nyc mocha"
+  },
+  "nyc": {
+    "reporter": [
+      "html",
+      "text"
+    ]
```

Comment:
Can you change the script like [this](https://github.com/rdf-ext/rdf-store-fs/blob/master/package.json#L8)? You are already close to what I plan to have as my minimal boilerplate standard. Hopeful soon I will find time to extend [thewolf](https://github.com/bergos/thewolf) to take care of this :-)

Reply:
Sure. If your intent is merely to move the flags inline, are you ok with also showing the coverage after running tests on the command line, either via the additional "text" reporter as I had or with the "text-summary" reporter, e.g., `"test": "standard && nyc --reporter=lcov --reporter=text mocha"`?

Reply:
Yes, with flags inline. I used only the `lcov` reporter which also generates html. So it's up to you.

Reply:
Ok, added, thanks.